contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-rule.y

```diff
@@ -1511,6 +1511,83 @@ table_ref:	relation_expr tsql_table_hint_expr
 					n->relation = (Node *) $1;
 					$$ = (Node *) n;
 				}
+			| TSQL_APPLY func_table func_alias_clause
```

Comment:
Let's add a test when func_alias_clause is used and referenced by another filter/join condition.

Reply:
Added